letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -530,7 +542,19 @@ CheckPathPermissions() {
 UNLIKELY_EOF
 }
 
-if [ "$1" = "--le-auto-phase2" ]; then
+if [ "$1" = "--le-auto-phase2" -a "$DEPRECATED_OS" = 1 ]; then
+  # Phase 2 damage control mode for deprecated OSes.
+  # In this situation, we bypass any bootstrap or certbot venv setup.
+  shift 1
+
+  error "Your system is not supported by certbot-auto anymore, and so certbot cannot be installed or upgraded."
+  error "Please consult official Certbot site (https://certbot.eff.org/) to check other alternatives."
```

Comment:
If we point people here (which I think we should), I think we need to make sure we have something to tell these users.

Given the complexity of getting a new version of Python and OpenSSL and how few users are affected, I think we should probably just add a warning to the top of the CentOS and RHEL 6 instructions saying that only x86_64 systems are supported.

Reply:
You mean, in the website?

Reply:
Yeah. I created https://github.com/certbot/website/issues/512 and https://github.com/certbot/certbot/issues/7632 to track this and I think we should get that PR ready before releasing this code.